types/result.go

```diff
@@ -264,3 +265,21 @@ func ParseABCILogs(logs string) (res ABCIMessageLogs, err error) {
 	err = json.Unmarshal([]byte(logs), &res)
 	return res, err
 }
+
+var _, _ types.UnpackInterfacesMessage = SearchTxsResult{}, TxResponse{}
+
+// UnpackInterfaces implements UnpackInterfacesMessage.UnpackInterfaces
+func (s SearchTxsResult) UnpackInterfaces(unpacker types.AnyUnpacker) error {
+	for _, tx := range s.Txs {
+		err := types.UnpackInterfaces(tx, unpacker)
```

Comment:
It makes sense to handle this because in general it will be implemented. We could update the `Tx` interface but I don't think we want to impose that requirement more generally right now.